geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/filters/RowFilter.scala

```diff
@@ -9,7 +9,17 @@
 package org.locationtech.geomesa.index.filters
 
 trait RowFilter {
+
+  import RowFilter.FilterResult
+
+  @deprecated("replaced with filter")
   def inBounds(buf: Array[Byte], offset: Int): Boolean
+
+  // TODO remove default impl in next major version
```

Comment:
Is it worth tagging a ticket for 4.0.0?

Reply:
added it to https://geomesa.atlassian.net/browse/GEOMESA-2966